testData/pkg/apis/fun/v1alpha1/toy_types.go

```diff
@@ -79,6 +82,9 @@ type ToySpec struct {
 	// Using this for testing purpose.
 	Rook *intstr.IntOrString `json:"rook"`
 
+	// +kubebuilder:validation:AnyOf[]=type:boolean,format:ipv6;type:number,format:ipv4
+	Server *BoolOrInt `json:"server,omitempty"`
```

Comment:
not sure this format makes sense here -- the format directive only applies for strings.

Reply:
My bad. I changed the code to make the format value more relevant.